### PR TITLE
fix: renaming the package and some minor ui cleanup

### PR DESCRIPTION
**Pull request by dshukertjr**
Thanks @FatumaA again for all the amazing work. The widgets that you implemented are very intuitive and easy to work with! I wanted to tweak the widgets a bit so that the border style will inherit the user's theme. Also wanted to change the package name if it was okay with you!

### CODE REVIEW

File: lib/src/screens/supa_email_auth.dart [hunk 1]
```diff
@@ -53,8 +54,7 @@ class _SupaEmailAuthState extends State<SupaEmailAuth> {
               return null;
             },
             decoration: const InputDecoration(
-              icon: Icon(Icons.email),
-              border: OutlineInputBorder(),
```
Comment (dshukertjr) [inline lib/src/screens/supa_email_auth.dart:57]: By removing the border property here, the border style will match what is set in the theme of MaterialApp!

File: lib/src/screens/supa_email_auth.dart [hunk 2]
```diff
@@ -70,93 +70,87 @@ class _SupaEmailAuthState extends State<SupaEmailAuth> {
...
-                            backgroundColor: Colors.red,
-                          ),
-                        );
-                      },
-                    );
-                    if (!mounted) return;
-                    Navigator.popAndPushNamed(
-                        context, widget.redirectUrl ?? '');
-                  }
+          ElevatedButton(
```
Comment (dshukertjr) [inline lib/src/screens/supa_email_auth.dart:82]: I have removed the surrounding SizedBox and used `crossAxisAlignment: stretch` on the parent column.

**Comment by FatumaA** [comment]
@dshukertjr looks good 👌 also, yeah the naming was a bit awkward 😅

**Review by FatumaA — APPROVED** [review]

**Comment by FatumaA** [comment]
Now that we have the email flow and the social sign ins, I was thinking that it is ok for a version 0.0.1 and we can add phone auth a bit later. 
So that I take the time to clean up the ui code and figure out the redirect matter. As well any other thing.  
What do you think?

**Comment by dshukertjr** [comment]
@FatumaA Yup, I agree! Could I make it 0.0.1-dev.1 so that we can introduce breaking changes in the future if we need to without feeling too bad?

**Pull request merged**